README.md

```diff
@@ -337,77 +335,54 @@ content is in an inline script file, then the script tag could be returned.
 
 If you'd rather disable this behavior, set `ignore` to `false`.
 
-### `getByAltText`
```

Comment:
Why are these doc sections being deleted?

Reply:
@alexkrolick refer to https://github.com/kentcdodds/dom-testing-library/pull/169#discussion_r240464612